src/modm/architecture/interface/assert.hpp.in

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2016-2018, 2020, Niklas Hauser
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include "assert.h"
+#include <modm/architecture/interface/register.hpp>
+/// @endcond
```

Comment:
On purpose, there is an issue somewhere where a `@code` hasn't been closed, but I couldn't find it in any of the header files. (at least none that were included here).